src/USER-MISC/angle_fourier_simple_approx.cpp

```diff
@@ -0,0 +1,218 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Tim Bernhard (ETHZ)
+------------------------------------------------------------------------- */
+
+#include "angle_fourier_simple_approx.h"
+#include <cmath>
+#include "atom.h"
+#include "comm.h"
+#include "force.h"
+#include "neighbor.h"
+
+
+#include "suffix.h"
+using namespace LAMMPS_NS;
+
+#define SMALL 0.001
+#define TWO_PI 2.*M_PI
+
+
+typedef struct { double x,y,z; } dbl3_t;
+typedef struct { int a,b,c,t;  } int4_t;
+
+/* ---------------------------------------------------------------------- */
+
+AngleFourierSimpleApprox::AngleFourierSimpleApprox(class LAMMPS *lmp)
+  : AngleFourierSimple(lmp)
+{
+  suffix_flag |= Suffix::OMP;
+}
+
+/* ---------------------------------------------------------------------- */
+
+/**
+ * Approximation of cos
+ * maximum error is about 0.00109 for the range -pi to pi
+ * Source: https://stackoverflow.com/a/28050328/3909202
+ */
+double AngleFourierSimpleApprox::fastCos(double x) {
+  constexpr double tp = 1./(TWO_PI);
```

Comment:
For improved readability, please use `invtwopi` pr `inv2pi` instead of `tp` and `1.0` / `0.25` instead of `1.` / `.25` and equivalent.